docs/examples/regex-check-for-mobile.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: default
+title: Check email for Mobile Number
+nav_order: 13
+parent: Examples
+---
+
+# Introduction
+We're going to be creating two workflows one to send an email with a mobile number inside. The second workflow will read the email body and use regex grab the mobile number. Which will lead to us sending a text message saying we are out of office using the 'twilio' container.
+
+## Send Email and Trigger Event
+
+```yaml
+id: send-mobile-trigger-event
+functions:
+- id: smtp
+  image: vorteil/smtp:v3
+- id: request
+  image: vorteil/request:v5
+description: This workflow sends an email and triggers an event.
+states:
+  # continued in next code block
+```
+
+
+### Send Email
+This action state uses the smtp container to send an email.
+
+```yaml
+- id: sendemail
+  type: action
+  action:
+    function: smtp
+    secrets: ["EMAIL_NAME", "EMAIL_PASSWORD"]
+    input:
+      to: "{{.secrets.EMAIL_NAME}}"
+      subject: "Your Review"
+      message: "Hello my name is Trent Hilliam please msg me on +61435545810."
```

Comment:
You real number is here: If you want to get rid of it, you can probably remove it in a commit and then squash these commits on merge.